### PR TITLE
feat: add article

### PR DESCRIPTION
**Pull request by craigbilner**
`author-profiles` and `topics` are already on SSR but `article` was outstanding. Adding for ease of development going forward and refactoring of render in the future

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/f0a80613df89516950b31302525d1aaccbca9cc0/index.html

**Comment by colincclark** [comment]
I don't see the issue for adding SSR testing, but we should start testing this package now that more and more things will rely on it?

**Comment by craigbilner** [comment]
@colincclark looking to address testing with [cypress](https://www.cypress.io/)

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/017c0b110251da038a55f0e014942d716ee4c13c/index.html

File: packages/ssr/ad-config.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "kuid": "L3uO9V7m",
+    "ksg":
+      "rzxmnyauv,sxzr5crda,pev88aflc,rs3e69b9t,qlrbrdisn,rqwt3l0vs,sqmyjv5y9,rs3w2whbv,rs2085or4,rw2q4y17n,rzxih7667,scd1httgm,rzxl9c9fc,rpwvy2tn3,oqq16c2eg,r3ge6ar30,szocxchms,rwusiipqe,scdzf5ue6,rzxlpwpj6,rpciv6gcv,scdy6k7fb,rpwr6s9yr,rpwxlidyq,rpwu9ul78,sx3y60mw4,scd369xa5,scd078y6q,q46ihbha4,sx3v2r3pl,p3cjf0mwf,rzxlzfg0s,rpwn918i6,rs269dhm1,sm60t5hdn,sat5wladc,scdyr0lr3,sat5h0dgj,sx3qr25yt,rs0a3n41s,sx3p6kpap,rzxmft3c7,rpwrel6lh,sau8ybfqv,rwuo9cp64,rpwtrhrep,scdxdxpny,sa158303k,sx3vrj8rn,rpwsw2wub,savahmdfp,rzxit9roh,szocq7e5b,sx3xy0rye,scd26efxg,scd4rp3dr,rzxmboyaj,sat4w5zr7,sa16lfkyn,rs20gu0ke,sau87bmgh,sau97llqr,sat6hypaq,sx3ysbbbg,rs22revlt,rv2fvbuyb,sx3wx1na1,sat6qec0x,scd561zdz,rwusbvbt6,rzxlxc8x9,sa2zih6pd,rpwunrlt9,scdye0ajw,sx3u2t0e7,rzxl2zwob,rs3gjttqi,sg1ia8twm,rsz87n468,pw58k5bt6,sx3oafly1,rzxj7jvxu,scd5qqxhc,sat55dl3g,rwuslm2uu,rsy610y7q,rwusge2br,sx3rsty6b,rs2zo0n9z,qu9h151f0,r6ddwq5od,rwusduoa8,sat4jqdso,rpww778sp,rz1gxal44,sx3sprq73,rzxmjser7,rs28i1y2j,rzxl5rcy4,rs238l43d,rpwubont7,sx3oy1hl6,sqmye2ubk,scmxb0crl,sau9itubd,scd6cqj7w,szoco0ngh,sx3navh0o,sa2vm2scg,rs3ct5unb,scd1rcsns",
+    "cips": null,
```
Comment (colincclark) [inline packages/ssr/ad-config.json]: https://github.com/newsuk/times-components/pull/1351/files#diff-7e6061767508b1d225e4ac993f59f5d6L23 this was removed here, is it needed?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/368aee16417f09af9dc0fc3d472def111fcc6f9a/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/7b4f2f8fc377cec8c34141866cc3f71f5f640244/index.html

File: packages/ssr/ad-config.json (hunk 1 above)
Comment (craigbilner) [reply packages/ssr/ad-config.json]: took this from the `article` package, will do a cleanup separately :+1:

**Pull request merged**